### ISSUE TITLE
Add thourough code docs

### ISSUE THREAD

**Issue by verity0001** [Riz-ve/Xeno#53]
Personally, I would jump onto this project if it had the necessary documentation, I don't want to go and spend entire days of my life trying to figure out what the code means or what it does. Write some docs, make a list of goals, and wait for people to open pull requests. Obviously a suggestion for the dev, they don't have to this, I personally think it just would be better. Great job so far.

**Comment by maximusloltvink** [comment]
I agree, this would help many people, including me, for example, at the moment I want to understand which attach method is used

**Comment by LuauDev** [comment]
just, just die

**Comment by LuauDev** [comment]
> I agree, this would help many people, including me, for example, at the moment I want to understand which attach method is used

maybe use ur brain and see which one

**Comment by Stratxgy** [comment]
> just, just die

obviously since youre saying this on a coding platform youre not mature enough to be here.

**Comment by rhuda21** [comment]
i also agree with this, it would help us understand how we can modify it and use it. ofc following licence. The disadvantage is tho more people are going to skid of it.

**Comment by Stratxgy** [comment]
> > > just, just die
> > 
> > 
> > obviously since youre saying this on a coding platform youre not mature enough to be here.
> 
> dude i was joking, its good thing to add a wiki, nerd

guys luaudev called me a nerd
